<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientInfoDialog</class>
 <widget class="QDialog" name="PatientInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ---- 对话框整体样式 ---- */
QDialog {
    background-color: #f5f7fa; /* 与主窗口一致的柔和背景色 */
}

/* ---- 顶部头像区域 ---- */
QLabel#photoLabel {
    border: 2px solid #ddd;
    border-radius: 40px; /* 圆形头像 */
    min-width: 80px;
    min-height: 80px;
    max-width: 80px;
    max-height: 80px;
}
/* 顶部的姓名和年龄输入框，我们让它看起来不像输入框 */
QLineEdit#nameLineEdit, QLineEdit#ageLineEdit {
    font-size: 18px;
    font-weight: bold;
    color: #2c3e50;
    border: none; /* 去掉边框 */
    background-color: transparent; /* 透明背景 */
}
QLineEdit#ageLineEdit {
    font-size: 14px;
    font-weight: normal;
    color: #555;
}


/* ---- 卡片样式 (QGroupBox) ---- */
QGroupBox {
    background-color: white;
    border: 1px solid #e0e0e0;
    border-radius: 10px;
    margin-top: 15px; /* 让标题和上一个元素有距离 */
    padding: 10px;
}

/* 卡片标题的样式 */
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 15px;
    padding: 0 5px 5px 5px;
    color: #3498db; /* 标题用主题蓝色 */
    font-weight: bold;
}

/* ---- 卡片内部的输入框样式 ---- */
QLineEdit {
    background-color: #f5f7fa; /* 浅灰色背景 */
    border: 1px solid #ddd;
    border-radius: 5px;
    padding: 5px;
    font-size: 13px;
}
/* 当输入框变为只读时，让它看起来更像普通文本 */
QLineEdit:read-only {
    background-color: white;
    border: none;
}


/* ---- 底部按钮样式 ---- */
QPushButton {
    min-width: 80px;
    min-height: 25px;
    border-radius: 8px;
    font-size: 13px;
}
/* 保存按钮用主题色 */
QPushButton#editSaveButton {
    background-color: #3498db;
    color: white;
    font-weight: bold;
}
QPushButton#editSaveButton:hover {
    background-color: #2980b9;
}
/* 关闭按钮用普通样式 */
QPushButton#closeButton {
    background-color: #bdc3c7;
    color: white;
}
QPushButton#closeButton:hover {
    background-color: #95a5a6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,3,3,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="photoLabel">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>84</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>84</width>
           <height>84</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220)</string>
         </property>
         <property name="text">
          <string>清空文本</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>姓名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>年龄</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ageLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>联系方式</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>501</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>手机号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="phoneLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>邮箱</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="emailLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>基本信息</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>501</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>出生年月</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>身份证号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dobLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="idCardLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="editSaveButton">
         <property name="text">
          <string>修改资料</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
